sdk/messaging/azservicebus/sender.go

```diff
@@ -199,7 +238,20 @@ func (sender *Sender) createSenderLink(ctx context.Context, session amqpwrap.AMQ
 	return amqpSender, nil, nil
 }
 
+func (s *Sender) startSpan(ctx context.Context, spanName string, operationName tracing.MessagingOperationName, attrs ...tracing.Attribute) (context.Context, func(error)) {
+	spanName = fmt.Sprintf("Sender.%s", spanName)
+	attributes := []tracing.Attribute{
```

Comment:
Maybe we should define these all the operation constants as 'Sender.ScheduledMessages' (ie, preformatted) and avoid the formatting operation/concat that we have to do here. That's easily for a future PR, I'm sure there's other spots I do the same thing.

Reply:
Just to clarify, did we want to define the span names when we are calling the span, or did we want to have a set of pre-defined span names (similar to the operation name/type below) where we can directly grab and use?

Reply:
"When calling" is fine, but having predefined 'const''s could be nice from a documentation perspective (ie: these are all the spans that exist).